compiler/rustc_error_codes/src/error_codes/E0137.md

```diff
@@ -1,24 +0,0 @@
-More than one function was declared with the `#[main]` attribute.
```

Comment:
Please don't remove it. It's important to keep track of the old error code explanations. Look at other removed code error explanations to see how it's done.

Reply:
Got it, thanks!